<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompilerNASMWidget</class>
 <widget class="QWidget" name="CompilerNASMWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NASM</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="4">
    <widget class="QCheckBox" name="chkLinkStdlib">
     <property name="text">
      <string>Link Standard System Startup Files and Libraries</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Path to NASM:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btnBrowserNASM">
     <property name="toolTip">
      <string>Browse</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="lblNASMInfo">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnTestNASM">
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtNASMPath"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtNASMPath</tabstop>
  <tabstop>btnBrowserNASM</tabstop>
  <tabstop>btnTestNASM</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
